src/main/java/com/aquent/viewtools/PostTool.java

```diff
@@ -88,9 +88,9 @@ public Credentials createCreds(String user, String pass) {
    * @param creds - A credentials object for authenticated requests.
    * @return A PostToolResponse Object
    */
-  public PostToolResponse sendStringData(String url, String data, String method, String contentType, Credentials creds) {
+  public PostToolResponse sendStringData(String url, String data, String method, String contentType) {
```

Comment:
we don't want to remove the creds/params from the method signatures.  only the logging statements